scrapers/stackoverflow.js

```diff
@@ -0,0 +1,61 @@
+var unirest = require("unirest");
+var question = "";
+for (i = 2; i < process.argv.length; i++) {
+    question += process.argv[i] + " ";
+}
+var url = "https://api.stackexchange.com/2.2/search/advanced?key=jAAMPr3uTSgYulMJ3LY5Hw((&order=desc&sort=activity&q=" + question + "&answers=1&site=stackoverflow";
```

Comment:
We can not scrape. Just check https://stackoverflow.com/robots.txt. Its not allowed. API is the only way. I willl remove api key and handle error cases.